pandas/io/excel.py

```diff
@@ -285,7 +311,7 @@ def read_excel(io,
                header=0,
                names=None,
                index_col=None,
```

Comment:
what happened to ``usecols``??

Reply:
Actually, it was never there. I forgot to add `usecols=None` in some functions because the code worked without it. =p